src/routers/companyRouter.ts

```diff
@@ -8,6 +8,7 @@ import { logger } from "../lib/Logger";
 import FormValidator from "../utils/formValidator.util";
 import CompanyNumberSanitizer from "../utils/companyNumberSanitizer";
 import * as constants from "../constants/app.const";
+import * as validationConstants from "../constants/validation.const";
```

Comment:
I think instead of pulling in all constants, we should only pull in what we require:

```suggestion
import { COMPANY_PROFILE, COMPANY_NUMBER } from "../constants/validation.const";
```